### ISSUE TITLE
Showing negative numbers in execution summary

### ISSUE THREAD

**Issue by sguptatw** [getgauge/gauge#439]
_From @manupsunny on June 1, 2016 6:45_

When adding gauge-java as file dependency instead of maven, the execution summary shows negative numbers. I think this happens because some dependencies of gauge-java are missing.

### Expected behavior
When any of the dependencies are missing it should prompt the user to add those dependencies

### Actual behavior
* Getting wrong execution summary. Example : 

```
Sucessfully generated Html-report to => <path>/sample/reportsCI/html-report
Specifications: -5 executed     -5 passed       0 failed        5 skipped
Scenarios:      0 executed      0 passed        0 failed        10 skipped
```

* Getting a report with wrong results. Example :
```
Congratulations! You've gone all green and saved the environment!
```

### Steps to reproduce
1. Copy `gauge-java-0.4.0.jar` to `libs` folder
2. Add `compile fileTree(dir: 'libs', include: ['*.jar'])` to dependencies in `build.gradle`
3. Execute` gauge` command

### Gauge version
Gauge version: 0.4.0
html-report (1.1.0)
java (0.3.3)

_Copied from original issue: getgauge/gauge-java#55_

**Comment by sguptatw** [comment]
This happens even when the BeforeSuite hook fails

**Comment by apoorvam** [comment]
This is fixed in case `before suite` hook fails. Fix available in nightly >= 02.06.2016

**Comment by kaustavdm** [comment]
This works now.
